inference-engine/tests/functional/plugin/shared/src/behavior/invalid_cases/proposal.cpp

```diff
@@ -93,6 +93,12 @@ void ProposalBehTest::SetUp() {
     function = std::make_shared<ngraph::Function>(results, params, "proposal");
 }
 
+void ProposalBehTest::Run() {
+    LoadNetwork();
+    Infer();
+    Validate();
```

Comment:
Is it necessary to use `Validate()` method in behavior tests?

Reply:
No. Removed